.mage/magefile.go

```diff
@@ -25,15 +27,36 @@ import (
 	"go.einride.tech/mage-tools/targets/mggoreview"
 )
 
+func init() {
+	mgmake.GenerateMakefiles(
+		mgmake.Makefile{
+			Path:          mgpath.FromGitRoot("Makefile"),
+			DefaultTarget: All,
+		},
+		mgmake.Makefile{
+			Path:          mgpath.FromGitRoot("proto/Makefile"),
+			Namespace:     Proto{},
+			DefaultTarget: Proto{}.Hej,
```

Comment:
What happens if the user provides `Proto.Hej` instead of `Proto{}.Hej`?

Reply:
`Cannot use 'Proto.Hej' (type func(Proto)) as the type func()`

Reply:
Change the type to `interface{}`? This won't work if the target returns `error` or takes a `context.Context` anyway?

Reply:
Sounds good :+1: